data/navigation_dataset/examples/objects_in_groups/example_0048.xml

```diff
@@ -0,0 +1,164 @@
+    <object name="barrel2" object_type="barrel" object_color="red" position="4.59777 -3.2152 0.270293" orientation="0,0,0,1"/>
+    <object name="barrel3" object_type="barrel" object_color="red" position="3.67026 -3.63568 0.270293" orientation="0,0,0,1"/>
+    <object name="chair0" object_type="chair" object_color="white" position="-2.13121 4.94816 0.01" orientation="0,0,0,1"/>
+    <object name="chair1" object_type="chair" object_color="white" position="-0.685016 4.16026 0.01" orientation="0,0,0,1"/>
+    <object name="chair2" object_type="chair" object_color="white" position="0.933675 4.03338 0.01" orientation="0,0,0,1"/>
+    <object name="chair3" object_type="chair" object_color="white" position="2.66517 4.6839 0.01" orientation="0,0,0,1"/>
+    <object name="cone0" object_type="cone" object_color="orange" position="-6.81 -5.13934 0.00962007" orientation="0,0,0,1"/>
+    <object name="cone1" object_type="cone" object_color="orange" position="-6.92304 -2.64793 0.00962007" orientation="0,0,0,1"/>
+    <object name="cone2" object_type="cone" object_color="orange" position="-7.90864 0.418486 0.00962007" orientation="0,0,0,1"/>
+    <object name="cone3" object_type="cone" object_color="orange" position="-6.5179 1.65635 0.00962007" orientation="0,0,0,1"/>
+    <object name="cone4" object_type="cone" object_color="orange" position="-6.88026 5.58281 0.00962007" orientation="0,0,0,1"/>
+    <object name="ground" object_type ="ground" object_color="na" position="0 0 0" orientation="0,0,0,1"/>
+    <object name="husky_mm" object_type="robot" object_color="na" position="0.369435 -0.010466 0.38539" orientation="0,0,0,1"/>
+  </world>
+  <NP>
+    <grounding_set>
+      <container container_type="row">
+        <object name="chair1" object_type="chair" object_color="white" position="-0.685016 4.16026 0.01" orientation="0,0,0,1"/>
+        <object name="chair2" object_type="chair" object_color="white" position="0.933675 4.03338 0.01" orientation="0,0,0,1"/>
+      </container>
+      <object name="chair1" object_type="chair" object_color="white" position="-0.685016 4.16026 0.01" orientation="0,0,0,1"/>
+      <object name="chair2" object_type="chair" object_color="white" position="0.933675 4.03338 0.01" orientation="0,0,0,1"/>
+    </grounding_set>
+    <NP>
+      <grounding_set>
+        <container container_type="row">
+          <object name="chair1" object_type="chair" object_color="white" position="-0.685016 4.16026 0.01" orientation="0,0,0,1"/>
+          <object name="chair2" object_type="chair" object_color="white" position="0.933675 4.03338 0.01" orientation="0,0,0,1"/>
+        </container>
+        <object name="chair1" object_type="chair" object_color="white" position="-0.685016 4.16026 0.01" orientation="0,0,0,1"/>
+        <object name="chair2" object_type="chair" object_color="white" position="0.933675 4.03338 0.01" orientation="0,0,0,1"/>
```

Comment:
Not sure if it is possible to resolve the objects since this is an independent phrase and does not have the container as context.